profile/rwahs.xml

```diff
@@ -925,11 +925,19 @@
         </label>
       </labels>
       <items>
-        <item idno="Photographer" enabled="1" default="0">
+        <item idno="Artist" enabled="1" default="0">
```

Comment:
Sorted these alphabetically and added elements required by public memorial form.